src/js/services/onGoingProcess.js

```diff
@@ -30,8 +30,16 @@ angular.module('copayApp.services').factory('ongoingProcess', function($log, $ti
     'deletingWallet': gettext('Deleting Wallet...'),
   };
 
+  root.isProcessing = false;
```

Comment:
I 'm not sure if it 's the best way to block some function in the controller walletHome, like a `loading` action or something. I just wanted to prevent this line: 

```
-    if (form.address.$invalid && !ongoingProcess.get('fetchingPayPro')) {
 +    if (form.address.$invalid && !ongoingProcess.isProcessing) {
```

If this don't like it, I agree to leave it as it was until we find another solution.